src/index.js

```diff
@@ -24,17 +27,58 @@ const fetchData = () => {
       tripsData: dataSet[1].trips,
       destinationsData: dataSet[2].destinations, 
     }).then(dataSet => {
-      travelers = new AllTravelers(dataSet.travelersData); 
-      //traveler instantiation
-      //agent instantiation 
+      allTravelers = new AllTravelers(dataSet.travelersData, dataSet.tripsData, dataSet.destinationsData); 
+      traveler = new Traveler(2, dataSet.travelersData, dataSet.tripsData, dataSet.destinationsData);
+      domUpdates.displayWelcome(traveler);
+      domUpdates.displayTravelersTrips(traveler);
```

Comment:
There is a lot going on inside of this .then statement, some of it could live in a function